Live/Classes/Food.h

```diff
@@ -0,0 +1,18 @@
+#ifndef __FOOD__H__
+#define __FOOD__H__
+
+#include "Item.h"
+
+#define FOOD_X 36
+#define FOOD_Y 1
+#define FOOD_WEIGHT 6
+
+class Food : public Item {
+public:
+    Food(int sprite_x, int sprite_y) :
+    Item("Spritesheet/roguelikeChar_transparent.png", FOOD_X, FOOD_Y, sprite_x, sprite_y, FOOD_WEIGHT) {}
```

Comment:
nonononno, even if it's temporary, either get a relevant spritesheet in the constructor, or put it in a macro.